### ISSUE TITLE
Opacity value with hover method

### ISSUE THREAD

**Issue by brollb** [DmitryBaranovskiy/g.raphael#191]
Hello,

I believe the value for 'opacity'  (on line 272) should be 0 rather than 1.

Brian

**Comment by vundicind** [comment]
I agree with you, why the opacity is set to '1'? I think it should be '0' like on line 238
